src/views/bootablevolumes/list/hooks/useBootableVolumesColumns.ts

```diff
@@ -97,7 +110,7 @@ const useBootableVolumesColumns = (
 
   const [activeColumns] = useKubevirtUserSettingsTableColumns<K8sResourceCommon>({
     columnManagementID: DataSourceModelRef,
-    columns,
+    columns: canGetNode ? columns : columns.filter((column) => column.id !== 'node'),
```

Comment:
why do we need it?
i dont see a node column in this file

Reply:
Not sure right now. I was experimenting with something and probably forgot to remove some related stuff.